demos/burgers-isotropic.py

```diff
@@ -0,0 +1,274 @@
+# Burgers equation with Hessian-based mesh adaptation
+# ===================================================
+
+# Yet again, we revisit the Burgers equation example. This time, we apply a goal-oriented
+# isotropic mesh adaptation. The interesting thing about this demo is that, unlike the
+# `previous demo <./point_discharge2d-goal_oriented.py.html>`__ and its predecessor,
+# we consider the time-dependent case. Moreover, we consider a :class:`MeshSeq` with
+# multiple subintervals and hence multiple meshes to adapt.
+#
+# As before, we copy over what is now effectively boiler plate to set up our problem. ::
+
+from firedrake import *
+from animate.adapt import adapt
+from animate.metric import RiemannianMetric
+from goalie import *
+from goalie_adjoint import *
+import matplotlib.pyplot as plt
+
+from firedrake.__future__ import interpolate
```

Comment:
This is new to me. Is it a fix for #105? If so, would you mind fixing that issue more widely?

Reply:
I hope you didn't make a start on this @acse-ej321! I was getting annoyed by the warnings so I opened #129 :)

Reply:
The more demos, the better, so it's okay if we have several.

Reply:
I was referring to the fix for the interpolation warning :) the demo looks great!